PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+NOTE:
+- Please fill out the template below 
+
+## Did you read the instructions in the [WORKFLOW](https://github.com/wtfpython-web/wtfpython-web/wiki/Workflow) ?
+
+> Y/N
+
+##  Is your Python code styled according to the PEP8 code style convention ?
+
+>Y/N
+
+## Have you run the tests ?
+
+>Y/N
```

Comment:
Add another task in the template,
```
Have you followed commit guidelines at https://github.com/wtfpython-web/wtfpython-web/wiki/Commit-Guidelines
```

Also, make sure that you follow it too in your PR :)